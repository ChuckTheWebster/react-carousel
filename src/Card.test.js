import React from "react";
import { render } from "@testing-library/react"
import Card from "./Card";

//TODO: Ask convention for organizing tests? Always have describe?
describe("render without crashing", function () {
  it("renders without crashing", function() {
    render(<Card caption="what" src="./photos/image1.jpg" currNum={1} totalNum={3}/>)
  });
});

describe("snapshot tests for stability", function () {
  it("matches snapshot", function () {
    const { container } = render(<Card caption="what" src="./photos/image1.jpg" currNum={1} totalNum={3}/>);
    expect(container).toMatchSnapshot();
  });
});