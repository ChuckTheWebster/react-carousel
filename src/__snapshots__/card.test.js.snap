// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`snapshot tests for stability matches snapshot 1`] = `
<div>
  <div
    class="Card"
  >
    <h4
      class="Card-title"
    >
      what
    </h4>
    <img
      alt="what"
      class="Card-image"
      src="./photos/image1.jpg"
    />
    <small
      class="Card-small"
    >
      Image 
      1
       of 
      3
      .
    </small>
  </div>
</div>
`;
